### DISCUSSION TITLE
You stored accesss token in sharedprefrances not in datastore, WHY?

### DISCUSSION THREAD

**Discussion by mohamedebrahim96** [PatilShreyas/NotyKT#134] [noty-android (Android Questions)] [answered]
@pranaypatel512 @PatilShreyas

**Comment by PatilShreyas** [comment, marked as answer]
@mohamedebrahim96 good question. So we have used `EncryptedSharedPreference` which allows us to store key and value pairs that can't be easily decrypted by penetrating. That's the main reason behind this.

**Reply by mohamedebrahim96** [reply]
thanks alot for your answer!
